internal/common/compilation.bzl

```diff
@@ -329,3 +374,12 @@ def ts_providers_dict_to_struct(d):
     if type(value) == type({}):
       d[key] = struct(**value)
   return struct(**d)
+
+def collect_js(ctx, deps):
+  """Aggregates transitive JavaScript source files from unfurled deps."""
```

Comment:
I have no idea what it means to unfurl a dep :)

Reply:
Removed!